cleanup.sh

```diff
@@ -5,14 +5,14 @@ echo "--Stopping the Open Event server--"
 
 if [ ! -z $OES_CONTAINER_ID ]; then
     cd open-event-server
-    docker compose down -v
```

Comment:
As we have discussed.   You *cannot* assume the dev / contributor have one or the other.   

Make sure the script tests and work for both!